docs/COMPILE-NIX.md

```diff
@@ -61,9 +61,14 @@ To build Unicorn on *nix (such as MacOSX, Linux, *BSD, Solaris):
 
         $ sudo ./make.sh install
 
-  For FreeBSD/OpenBSD, where sudo is unavailable, run:
+  For OpenBSD, where sudo is unavailable, run:
 
         $ su; ./make.sh install
+        
+  On FreeBSD, system's `make` is different from GNU make, so you need to install it.
+  
+        # pkg install gmake
+        $ gmake
```

Comment:
rather than update this part on FreeBSD to ask users to directly use `gmake`, a better solution is to fix `make.sh` for FreeBSD. if you can fix this, please send a pull req.